dev/io.openliberty.microprofile.telemetry.internal.common/src/io/openliberty/microprofile/telemetry/internal/common/rest/jakarta/TelemetryClientAsyncTaskWrapper.java

```diff
@@ -54,14 +54,4 @@ public <T> Callable<T> wrap(Callable<T> c) {
         }
     }
 
-    @Override
-    public <T> Supplier<T> wrap(Supplier<T> s) {
-        try {
-            return Context.current().wrapSupplier(s);
-        } catch (Exception e) {
-            Tr.error(tc, Tr.formatMessage(tc, "CWMOT5002.telemetry.error", e));
-            return s;
-        }
-    }
-
```

Comment:
This method should be put back in. It's more efficient than the default method on the interface.